tests/components/modbus/test_climate.py

```diff
@@ -170,6 +172,30 @@
                 }
             ],
         },
+        {
+            CONF_CLIMATES: [
+                {
+                    CONF_NAME: TEST_ENTITY_NAME,
+                    CONF_TARGET_TEMP: 117,
+                    CONF_ADDRESS: 117,
+                    CONF_SLAVE: 10,
+                    CONF_MIN_TEMP: 23,
+                    CONF_MAX_TEMP: 57,
+                }
+            ],
+        },
+        {
+            CONF_CLIMATES: [
+                {
+                    CONF_NAME: TEST_ENTITY_NAME,
+                    CONF_TARGET_TEMP: 117,
+                    CONF_ADDRESS: 117,
+                    CONF_SLAVE: 10,
+                    CONF_MIN_TEMP: -23,
+                    CONF_MAX_TEMP: -57,
```

Comment:
Having the max temp lower than the min temp doesn't make sense, maybe a check for that would be good?

Reply:
Not true, it would be fairly normal for a freezer.

Reply:
Just to be sure I was not being theoretical, I checked my refrigerator, it says "max: -19", and "min: -5", lthe current temp is "-7", so max lower than min is making sense for that one.

Reply:
Okay, then I haven't said anything 😅 Thanks for checking!

Reply:
The convention is that the negative number closest to 0 is the maximum number.
I've known this since lower school, a long time ago :)
And internet "says' the same, google it.
Don't rely on a single wrong occurrence please :)

Reply:
The thing is unfortunately that freezer logic is not neccessarily real math logic. The question is if we want to be strict and say that `max > min` should always be true.

Reply:
@autinerd good point, calculation-rules show that -23 is the max value.
Freezer logic does not exist, that's just a wrong implementation from someone that doesn't know the normal conventions.

Reply:
This is a nice theoretical discussion, but this PR allows min/max temperatures to be negative, not more not less.

Adding a check on min/max values (of which there are 7-8 different in modbus) is another PR...and I for one believe if a user wants to configure max lower than min, then he sure have reasons, and does it make HA less robust to allow that...

Reply:
But in order to avoid a heated discussion, I have changed the test (it really does not matter to the test), so that it now reads:
```
              CONF_MIN_TEMP: 23,
              CONF_MAX_TEMP: 57,
```
in the hope we can move on, and actually get the bug fixed, so the user who filed the issue can continue using his configuration.